Description: Fix BIGENDIAN issue in nxcomp (relevant on PPC64 arch)
Author: Mihai Moldovan <ionic@ionic.de>

--- a/nxcomp/Loop.cpp
+++ b/nxcomp/Loop.cpp
@@ -6832,9 +6832,9 @@
     {
       sockaddr_in newAddr;
 
-      size_t addrLen = sizeof(sockaddr_in);
+      socklen_t addrLen = sizeof(sockaddr_in);
 
-      newFD = accept(proxyFD, (sockaddr *) &newAddr, (socklen_t *) &addrLen);
+      newFD = accept(proxyFD, (sockaddr *) &newAddr, &addrLen);
 
       if (newFD == -1)
       {
